app/src/main/java/kr/co/connect/boostcamp/livewhere/ui/map/view/BackdropMotionLayout.kt

```diff
@@ -0,0 +1,62 @@
+package kr.co.connect.boostcamp.livewhere.ui.map.view
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.graphics.Rect
+import android.util.AttributeSet
+import android.view.MotionEvent
+import androidx.constraintlayout.motion.widget.MotionLayout
+import kotlinx.android.synthetic.main.activity_map.view.*
+import kr.co.connect.boostcamp.livewhere.R
+
+class BackdropMotionLayout(context: Context, attributeSet: AttributeSet? = null) : MotionLayout(context, attributeSet) {
+    private val viewRect = Rect()
+    private var touchStarted = false
+
+    init {
+        setTransitionListener(object : MotionLayout.TransitionListener {
+            override fun onTransitionTrigger(p0: MotionLayout?, p1: Int, p2: Boolean, p3: Float) {
+            }
```

Comment:
```suggestion
            override fun onTransitionTrigger(p0: MotionLayout?, p1: Int, p2: Boolean, p3: Float) = Unit
```
P3. 내부에 구현체가 없으면 Unit을 리턴해줘도 좋습니다